sc3-prepare.R

```diff
@@ -2,75 +2,114 @@
 # Script to creare SCE object for clustering with SC3.
 # Carlos Talavera-López - 2018
 
-# Load optparse we need to check inputs
-
-suppressPackageStartupMessages(require(optparse))
-
-# Load common functions
-
-suppressPackageStartupMessages(require(workflowscriptscommon))
+# Load required packages
+packages <- c("tidyverse", "SingleCellExperiment", "scater", "optparse", "SC3","workflowscriptscommon")
```

Comment:
With the exception of workflowscriptscommon (required for argument parsing), other packages should only be loaded once argument parsing is complete. Otherwise we waste time loading packages only to find arguments don't parse correctly.

Reply:
Also, please keep dependencies to a minimum. Tidyverse is probably not necessary.